pom.xml

```diff
@@ -16,7 +16,7 @@
         <main.basedir>${project.basedir}</main.basedir>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <hazelcast.version>5.5.0-SNAPSHOT</hazelcast.version>
+        <hazelcast.version>5.4.0</hazelcast.version>
```

Comment:
note: Release Pipeline will update master when we next do MAJOR or MINOR

Reply:
> note: Release Pipeline will update master when we next do MAJOR or MINOR

Are you sure it will update? I thought it only updated the current `master` `SNAPSHOT` version to new one (i.e. `5.5.0-SNAPSHOT` -> `5.6.-SNAPSHOT`), not from previously released to newly released?

Some of the files use older versions of Hazelcast and they haven't been updated.

Reply:
You are right and thats what I meant like this one https://github.com/hazelcast/hazelcast-code-samples/commit/3a963a683af992b325db8c41f52edef9b43d76cf. Release pipeline only updates <hazelcast.version> which should trickle down to sub modules 

So I am thinking if dependabot updates this and we make MAJOR/MINOR, will dependabot create a master PR, lets say we tag v5.6.0 in the future? I am sure it will but just want to check!

Reply:
> You are right and thats what I meant like this one [3a963a6](https://github.com/hazelcast/hazelcast-code-samples/commit/3a963a683af992b325db8c41f52edef9b43d76cf). Release pipeline only updates <hazelcast.version> which should trickle down to sub modules

I think it only updates it if it's the specific previous `SNAPSHOT` (or whatever version is specified when you runn it), so now here it will do nothing.
 
> So I am thinking if dependabot updates this and we make MAJOR/MINOR, will dependabot create a master PR, lets say we tag v5.6.0 in the future? I am sure it will but just want to check!

This change (I think!) makes release pipeline not affect the hazelcast version going forwards. And Dependabot will (and does) think Hazelcast is just-another-dependency and suggests updates as and and when there are any publically.

Reply:
I think you are right but if nothing is updated, then I think commit might fail as looks like currently `allowEmpty` is not passed and set to `false`. Worth checking!

Reply:
> I think you are right but if nothing is updated, then I think commit might fail as looks like currently `allowEmpty` is not passed and set to `false`. Worth checking!

Good point - are you able to amend as part of the other stuff your doing, please?

Reply:
yes sure. just remind in few days but I am sure I will remember